pkgs/servers/bililiverecorder/default.nix

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
Please format this file with nixpkgs-fmt.

Reply:
Done.